src/sentry/models/activity.py

```diff
@@ -85,6 +85,10 @@ class Meta:
 
     __repr__ = sane_repr('project_id', 'group_id', 'event_id', 'user_id', 'type', 'ident')
 
+    @staticmethod
+    def get_version_ident(version):
+        return (version or '')[:64]
```

Comment:
feel free to tell me this is dumb, i just didn't really wanna do `[:64]` in 6 places

Reply:
Nah, this is fine.